tests/test_completion.py

```diff
@@ -708,7 +708,7 @@ def check(
         )
         check(
             resolve_support=False,
-            expected_regex=r"^f\(X&amp; x\) \| does things$",
+            expected_regex=r"^f\(X&amp; x\)$",
             label="f",
             label_details={"detail": "(X& x)", "description": "does things"}
         )
```

Comment:
Those test are poor-mans version of the new ones so it would possibly be OK to remove those and convert to the new ones, if those test anything extra.

I say "poor-mans" version because those ignore "annotation" for example.